crates/api_crud/src/site/create.rs

```diff
@@ -41,31 +40,30 @@ impl PerformCrud for CreateSite {
   #[tracing::instrument(skip(context))]
   async fn perform(&self, context: &Data<LemmyContext>) -> Result<SiteResponse, LemmyError> {
     let data: &CreateSite = self;
-
+    let local_user_view = local_user_view_from_jwt(&data.auth, context).await?;
     let local_site = LocalSite::read(context.pool()).await?;
 
+    // BEGIN VALIDATION
```

Comment:
You can remove these BEGIN VALIDATION and END VALIDATION comments.

All the APIs run some validation logic, and it might be a good code cleanup task to move them into functions. Not necessary for this one tho.

Reply:
Removed comments and moved into a function!